### PR TITLE
Add es_ES translations

### PR DESCRIPTION
**Pull request by hugmanrique**
References for technical terms:
"connect to"≡"conectar a": https://www.rae.es/dpd/conectar
"kick"≡"echar": https://support.discord.com/hc/es/articles/213507137--Qu%C3%A9-es-Suprimir-C%C3%B3mo-lo-uso-""
"tab completion"≡"finalización con tabulación": https://docs.microsoft.com/es-es/powershell/scripting/windows-powershell/ise/how-to-use-tab-completion-in-the-script-pane-and-console-pane?view=powershell-7.1
"plugin"≡"complemento": https://docs.microsoft.com/es-es/powerapps/developer/data-platform/tutorial-write-plug-in
https://support.apple.com/es-es/guide/ipad/ipad213a25b2/ipados
https://support.apple.com/es-es/HT201400
https://filestore.community.support.microsoft.com/api/images/06079330-f6aa-4dc4-bfab-e62f9194a5bc?upload=true

The GNUv3 license name in Spanish was taken from https://www.gnu.org/licenses/licenses.es.html#GPL.

### CODE REVIEW

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_es_ES.properties [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+# Commands
+velocity.command.generic-error=Se ha producido un error al ejecutar este comando.
+velocity.command.command-does-not-exist=Este comando no existe.
+
+velocity.command.players-only=Solo los jugadores pueden ejecutar este comando.
+velocity.command.server-does-not-exist=El servidor {0} no existe.
+
+velocity.command.server-current-server=Estás conectado a {0}.
+velocity.command.server-too-many=Hay demasiados servidores registrados. Usa la finalización con tabulación para ver todos los servidores disponibles.
```
Comment (4drian3d) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_es_ES.properties:41]: "la finalización con tabulación" no tiene sentido en el contexto que se da. Para ello se puede usar mejor "el tabulador" o "el autocompletado" para referirse al Tab Complete, de lo contrario, si alguien tiene otra idea para expresarlo mejor, puede sugerirla.

Comment (hugmanrique) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_es_ES.properties:41]: As referenced above, that's how [Microsoft calls it in PowerShell](https://docs.microsoft.com/es-es/powershell/scripting/windows-powershell/ise/how-to-use-tab-completion-in-the-script-pane-and-console-pane?view=powershell-7.1), and given the console-like similarities of Minecraft chat commands that's what I found best fit this message. "Autocompletado" just means "autocompletion" and doesn't tell the user what action (pressing the Tab key) they need to take. "El tabulador" is "the tab" and would translate to "Use the tab to see..." (which doesn't mention pressing tab will autocomplete, the meaning of "seeing" is ambiguous imo). I agree the current message is a bit long, but it's the best I could find in use by a shell.

I did a 5 minute search and that's all I could find. The Spanish Wikipedia article for "Shell" (and others) doesn't mention this feature (unlike the English page 😕). Please share other references with different names and I will change the message. I don't have a Crowdin account, so I couldn't access the translation files for Minecraft, so if someone can share the translated message, we can use the official wording by Mojang.

**Comment by astei** [comment]
Can you update your translations with the new messages in 1e8454c7? Thanks!

**Comment by astei** [comment]
It looks like someone has finished another Spanish translation on [Crowdin](https://crowdin.com/project/velocity/es-ES). I invite you to review their work there.

Thanks for the contribution!

**Pull request closed without merging**